### PR TITLE
Modified Q

### PR DESCRIPTION
**Pull request by AndreyTepaykin**
1) Decreased Q/logs/removeAfterDays from 10 to 2. Logs use much disk space. We don't need so much logs.

2) Added platform/plugins/Streams/vendor/ to gitignore.

3) Fixed Q/placeholders. Make it disabled if field filled.

### CODE REVIEW

File: platform/plugins/Q/web/js/fn/placeholders.js [hunk 1]
```diff
@@ -114,7 +116,6 @@ function () {
 				'padding-bottom': props['padding-bottom'],
 				'border-top': 'solid ' + $this.css('border-top-width') + ' transparent',
 				'border-left': 'solid ' + $this.css('border-left-width') + ' transparent',
-				'box-sizing': $this.css('box-sizing'),
```
Comment (EGreg) [inline platform/plugins/Q/web/js/fn/placeholders.js:117]: why did you remove this one?

**Review by EGreg — APPROVED** [review]

**Comment by AndreyTepaykin** [comment]
Because it defined twice.
    On Thursday, November 28, 2019, 05:29:56 AM GMT+3, Gregory Magarshak <notifications@github.com> wrote:  
 
 
Merged #1449 into master.
